mlrun/api/crud/logs.py

```diff
@@ -144,7 +144,8 @@ async def _get_logs_from_logs_collector(
         size: int = -1,
         offset: int = 0,
     ) -> typing.AsyncIterable[bytes]:
-        async for log in log_collector.LogCollectorClient().get_logs(
+        log_collector_client = log_collector.LogCollectorClient()
```

Comment:
I was about to say that we might want to run this in a separate thread as it does some initialization like creating channels, but then looked inside the implementation and it seem to be creating async channels. So just throwing my thoughts here :)

Reply:
Also - we kept it as a singleton class, so it is only creating the async channel once.